code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -1,7 +1,52 @@
-#define CRYOMOBS 'icons/obj/cryo_mobs.dmi'
 ///Max temperature allowed inside the cryotube, should break before reaching this heat
 #define MAX_TEMPERATURE 4000
 
+/// This is a visual helper that shows the occupant inside the cryo cell.
+/atom/movable/cryo_occupant_vis
+	icon = 'icons/obj/cryogenics.dmi'
+	// Must be tall, otherwise the filter will consider this as a 32x32 tile
+	// and will crop the head off.
+	icon_state = "mask_bg"
```

Comment:
Can you put this on a general path for visual objects? Something like this: <https://github.com/tgstation/TerraGov-Marine-Corps/blob/cdd90c03d8833c36454686f01b02c8136bbf9eac/code/game/visual_objects.dm>

Reply:
Tbh this seems unnescessary. Even in the example you gave there is no shared logic apart from `mouse_opacity = MOUSE_OPACITY_TRANSPARENT`.

Reply:
The long-term idea is to group visual objects and start moving them away from `/obj/effects` where they are unfortunately held for the most part.